### PR TITLE
feat: Account settings operator

### PR DESCRIPTION
**Pull request by alainajaffer**
When the operator user clicks the “My Account” button

Then the operator user should only be presented with the following options:

- Account settings
- Social media
- Sign out

Given an operator user is within the “Account Settings” page

Then they should see their name (needs to be included in design) alongside noc code and fields as per wireframes

### CODE REVIEW

File: site/components/layout/Header.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ interface HeaderProps {
 const Header = ({ session, csrfToken }: HeaderProps): ReactElement => {
     const { buttonProps, itemProps, isOpen, setIsOpen } = useDropdownMenu(session?.isOrgAdmin ? 4 : 2);
 
+    console.log(session);
```
Comment (DeannaSharma) [inline site/components/layout/Header.tsx:18]: remove console.log :)

Comment (alainajaffer) [reply site/components/layout/Header.tsx]: done

File: site/schemas/operator.schema.ts
Comment (DeannaSharma) [inline site/schemas/operator.schema.ts:1]: Could we move this operator schema over to the organisations schema file? 

I like what you did with the transform so I implemented it in my add user PR, I added the below code to the organisations schema file:

```
export const operatorOrgSchema = subOrganisationSchema.transform((data) => ({
    orgId: data.PK,
    operatorOrgId: data.SK.replace("OPERATOR#", ""),
    name: data.name,
    nocCodes: data.nocCodes,
}));

export type OperatorOrgSchema = z.infer<typeof operatorOrgSchema>;

export const operatorOrgListSchema = z.array(operatorOrgSchema);
```

Comment (alainajaffer) [reply site/schemas/operator.schema.ts:1]: done

File: site/schemas/session.schema.ts [hunk 2]
```diff
@@ -11,19 +11,23 @@ export const sessionSchema = z
...
         "cognito:groups": z.array(z.nativeEnum(UserGroups)).optional(),
     })
     .transform((item) => {
         const isSystemAdmin = item["cognito:groups"]?.includes(UserGroups.systemAdmins) ?? false;
         const isOrgAdmin = item["cognito:groups"]?.includes(UserGroups.orgAdmins) ?? false;
         const isOrgPublisher = item["cognito:groups"]?.includes(UserGroups.orgPublishers) ?? false;
         const isOrgStaff = item["cognito:groups"]?.includes(UserGroups.orgStaff) ?? false;
+        const isOperatorUser = item["cognito:groups"]?.includes(UserGroups.operators) ?? false;
 
         return {
```
Comment (DeannaSharma) [inline site/schemas/session.schema.ts:24]: Might need some clarification on AC3 for this ticket which is about setting the datasource for operator users. 

At the moment, the datasource will be whatever the LTA org has set - if we follow the ACs then we will need some logic on line 45 to set the datasource for operator users

Comment (alainajaffer) [reply site/schemas/session.schema.ts:24]: done

**Review by DeannaSharma — APPROVED** [review]

**Pull request merged**